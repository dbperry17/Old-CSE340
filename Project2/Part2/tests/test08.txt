
Programming

LANGUAGE

programmingabc

programmingcse340

cse340 340language
