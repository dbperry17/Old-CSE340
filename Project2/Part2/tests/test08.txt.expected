10 ID language
10 NUM 340
10 ID cse340
